data/committees.js

```diff
@@ -487,7 +487,7 @@ export default [
 		class: 'cloud',
 		slug: 'cloud',
 		tagline: 'Building the Future in the Clouds',
-		external_link: null,
+		external_link: 'https://cloud.uclaacm.com/',
 		links: [
 			{
```

Comment:
can we add cloud website link (https://cloud.uclaacm.com/) in the links section here similar to how it's been done for other committees? thanks!